library/extractor/src/test/java/com/google/android/exoplayer2/extractor/mp4/Mp4ExtractorTest.java

```diff
@@ -102,4 +102,10 @@ public void mp4SampleWithColorInfo() throws Exception {
     ExtractorAsserts.assertBehavior(
         Mp4Extractor::new, "media/mp4/sample_with_color_info.mp4", simulationConfig);
   }
+
+  @Test
+  public void mp4SampleWithDolbyTrueHDTrack() throws Exception {
+    ExtractorAsserts.assertBehavior(
+        Mp4Extractor::new, "media/mp4/sample_dthd.mp4", simulationConfig);
```

Comment:
Would it be possible to reduce the size of the test file to less than 100KB (for example by reducing its duration)?

Reply:
The current test file represents 1s of audio.  For this test , a 0.4s teststream would certainly do..
I'll regenerate the stream and update the reference dumps.